src/scss/elements/_anchor.scss

```diff
@@ -0,0 +1,11 @@
+.anchor-btn{
+    background-color: #337099;
+    color: white !important;
```

Comment:
Do we need the `!important` line here?

Typically (not always, there are some rare instances where it is fine) `!important` should be avoided as a rule of thumb

Reply:
unfortunately, if I remove important, the font colour is black

Reply:
So the black colour is being applied by the pseudo class `visited`
```scss
a:visited {
    color: $pickled-bluewood;
}
```

So all generic `a` tags that have the visited state applied will apply the pseudo class `:visited`

As this is a button, we don't want that pseudo class styling to be applied, so we can add something like:

```scss
.anchor-btn {
    &:visited {
        color: white
    }
    background-color: #337099;
    color: white;
    display: inline-block;
    border-radius: 4px;
    padding: 15px 25px;
    font-size: 18px;
    text-decoration: none;
    max-width: 215px;
    min-width: 100px;
  }
```

This works as a class selector  (anything that starts with a `.` is more specific than a generic type selector such as just `a`

The `&` here just means copy parent name so as the parent of `&:visited` is `.anchor-btn` the `&` will be replaced when transpiled from SCSS to CSS as `.anchor-btn:visted`

Reply:
Although note #337099 should also be a variable `$ash` I believe matches that colour